src/components/MeetingPanel/index.js

```diff
@@ -0,0 +1,408 @@
+                    schedule: {
+                      ...meeting.schedule,
+                      durationInMinutes,
+                    }
+                  });
+                }} />
+            </div>
+          </div>
+        </div>
+      </Section>
+    );
+    const recurringMeeting = (
+      <Section className={styles.section}>
+        <div>
+          <div className={styles.spaceBetween}>
+            <span className={styles.label}>
+              {i18n.getString('recurringMeeting', currentLocale)}
+            </span>
+            <Switch
+              checked={isRecurring}
+              onChange={(isCheckRecurring) => {
+                const meetingType = isCheckRecurring ? 'Recurring' : 'Scheduled';
+                update({
+                  ...meeting,
+                  meetingType,
+                });
+              }} />
+          </div>
+          {
+            isRecurring ? (
+              <div className={styles.recurringDescribe}>
+                {i18n.getString('recurringDescribe', currentLocale)}
+              </div>
+            ) : null
+          }
+        </div>
+      </Section>
+    );
+    const video = (
+      <Section title={i18n.getString('video', currentLocale)} withSwitch>
+        <div>
+          <div className={classnames(styles.labelLight, styles.fixTopMargin)}>
+            {i18n.getString('videoDescribe', currentLocale)}
+          </div>
+          <div className={classnames(styles.spaceBetween, styles.fixTopMargin)}>
+            <span className={styles.labelLight}>
+              {i18n.getString('host', currentLocale)}
+            </span>
+            <Switch
+              checked={meeting.startHostVideo}
+              onChange={(startHostVideo) => {
+                update({
+                  ...meeting,
+                  startHostVideo,
+                });
+              }} />
+          </div>
+          <div className={classnames(styles.spaceBetween, styles.fixTopMargin)}>
+            <span className={styles.labelLight}>
+              {i18n.getString('participants', currentLocale)}
+            </span>
+            <Switch
+              checked={meeting.startParticipantsVideo}
+              onChange={(startParticipantsVideo) => {
+                update({
+                  ...meeting,
+                  startParticipantsVideo,
+                });
+              }} />
+          </div>
+        </div>
+      </Section>
+    );
+    const audioOptions = (
+      <Section title={i18n.getString('audioOptions', currentLocale)} withSwitch>
+        <CheckBox
+          onSelect={({ key }) => {
+            const audioOptions = key.split('_');
+            update({
+              ...meeting,
+              audioOptions,
+            });
+          }}
+          valueField={'key'}
+          textField={'text'}
+          selected={meeting.audioOptions.join('_')}
+          data={AUDIO_OPTIONS} />
+      </Section>
+    );
+    const meetingOptions = (
```

Comment:
Would you please split this render function to multiple components. It is too big.